internal/tools/aws/constants.go

```diff
@@ -53,17 +53,36 @@ const (
 	// Note: This needs to be manually created before RDS databases can be used.
 	DefaultDBSubnetGroupName = "mattermost-databases"
 
+	// DatabaseTypeMySQLAurora is a MySQL database running on AWS RDS Aurora.
+	DatabaseTypeMySQLAurora = "MYSQL/Aurora"
+
+	// DatabaseTypePostgresSQLAurora is a PostgreSQL database running on AWS
+	// RDS Aurora.
+	DatabaseTypePostgresSQLAurora = "PostgreSQL/Aurora"
+
+	// CloudInstallationDatabaseTagKey is the common tag key for determing
+	// database type.
+	CloudInstallationDatabaseTagKey = "tag:MattermostCloudInstallationDatabase"
+
 	// DefaultDBSecurityGroupTagKey is the default DB security group tag key
 	// that is used to find security groups to use in configuration of the RDS
 	// database.
 	// Note: This needs to be manually created before RDS databases can be used.
 	DefaultDBSecurityGroupTagKey = "tag:MattermostCloudInstallationDatabase"
```

Comment:
those 2 tags are same, do we want to use one of them, or it will cause problems down the line?

Reply:
Yeah, we have a couple duplicate tags already. Instead of cleaning them up here, I was going to do a follow-up ticket to take care of them.